src/Microsoft.XmlSerializer.Generator/tests/Microsoft.XmlSerializer.Generator.Tests.csproj

```diff
@@ -58,7 +58,7 @@
     <Message Text="Running Serialization Tool" Importance="normal" />
     <Exec Command="$(GeneratorCliPath)dotnet $(OutputPath)dotnet-Microsoft.XmlSerializer.Generator.dll $(OutputPath)Microsoft.XmlSerializer.Generator.Tests.dll /force /quiet"  />
     <Warning Condition="Exists('$(OutputPath)$(SerializerName).cs') != 'true'" Text="Fail to generate $(OutputPath)$(SerializerName).cs"/>
-    <Csc Condition="Exists('$(OutputPath)$(SerializerName).cs') == 'true' AND '$(MSBuildRuntimeType)' != 'core'" 
+    <Csc Condition="Exists('$(OutputPath)$(SerializerName).cs') == 'true'" 
```

Comment:
Is this going to work on desktop still? I assume we aren't always running on core here.

Reply:
This used to be the way we did it on desktop actually, we just had a special way of doing it for core because we had to pass in an extra property, but that is not required anymore since now we are actually using the task library from the core msbuild.

Reply:
OK so long as it works on both then I'm good.